src/middleware/github-server-app-middleware.ts

```diff
@@ -4,11 +4,17 @@ import { Installation } from "models/installation";
 
 export const GithubServerAppMiddleware = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 	const { jiraHost } = res.locals;
-	const { id } = req.params;
+	const { id: idStr } = req.query;
+	const id = parseInt(idStr as string);
+
+	if (id && isNaN(id)) {
+		req.log.warn({ id, jiraHost }, "Provided id found but not a number " + id);
```

Comment:
Don't add dynamic vars to the log, makes aggregation impossible. It's already part of the data.